### PR TITLE
Pipe docs

### PR DESCRIPTION
**Pull request by mhevery**

### CODE REVIEW

**Comment by vicb** [comment]
CI is red

File: modules/@angular/common/src/pipes/async_pipe.ts [hunk 1]
```diff
@@ -42,29 +42,27 @@ var _observableStrategy = new ObservableStrategy();
 var __unused: Promise<any>;  // avoid unused import when Promise union types are erased
 
 /**
+ * @ngModule CommonModule
+ * @whatItDoes Unwraps a value from an asynchronous primitive.
+ * @howToUse `observable_or_promise_expression | async`
```
Comment (vicb) [inline modules/@angular/common/src/pipes/async_pipe.ts]: event ?

File: modules/@angular/common/src/pipes/async_pipe.ts [hunk 2]
```diff
@@ -42,29 +42,27 @@ var _observableStrategy = new ObservableStrategy();
...
+ * @description
  * The `async` pipe subscribes to an `Observable` or `Promise` and returns the latest value it has
- * emitted.
- * When a new value is emitted, the `async` pipe marks the component to be checked for changes.
- * When the component gets destroyed, the `async` pipe unsubscribes automatically to avoid
+ * emitted. When a new value is emitted, the `async` pipe marks the component to be checked for
+ * changes. When the component gets destroyed, the `async` pipe unsubscribes automatically to avoid
```
Comment (zoechi) [inline modules/@angular/common/src/pipes/async_pipe.ts]: Should it be mentioned that it causes change detection for components with `ChangeDetectionStrategy.OnPush`?

**Comment by IgorMinar** [comment]
fixed up and merged as 5fad37d..2c42a50

**Pull request closed without merging**